src/Actions/Actions/DataManager.cs

```diff
@@ -91,20 +94,17 @@ internal void RemoveElementContext(Guid ecId)
         internal void RemoveDataContext(Guid ecId, bool keepMainElement = true)
         {
             // check whether key exists. if not, just silently ignore.
-            if (_elementContexts.ContainsKey(ecId))
+            ElementContext ec = GetElementContext(ecId);
```

Comment:
Quick note for this class. The old code repeated this pattern several times:
```
if (_elementContexts.ContainsKey(ecId))
{
  var ec = _elementContexts[ecId];
  if (ec != null)
  {
    // Do stuff with ec
  }
}
```

Since there's already a `GetElementContext` method, the new pattern is as follows:

```
ElementContext ec = GetElementContext(ecId);
if (ec != null)
{
  // Do stuff with ec
}
```

It's a tiny bit shorter, although it hides the fact that `GetElementContext` is connected to `_elementContexts`. I'm happy to entertain other approaches if you have any suggestions